### PR TITLE
chore: Upgrade Looker eslint and prettier config

### PR DESCRIPTION
**Pull request by bryans99**
Knock on effects:
1. eslint needed to be upgraded
2. prettier needed to be upgraded
3. new rules required quite a lot of code changes :(

In addition removed prettier and eslint configs from access-token-server package.json as not needed there.

### CODE REVIEW

File: packages/api-explorer/src/components/DocMethodSummary/utils.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ import React from 'react'
 import styled from 'styled-components'
 import { Icon, Status } from '@looker/components'
 import { Beaker } from '@looker/icons'
-import { VisibilityOff } from '@styled-icons/material'
+import { VisibilityOff } from '@styled-icons/material/VisibilityOff'
```
Comment (jkaster) [inline packages/api-explorer/src/components/DocMethodSummary/utils.tsx:30]: Interesting the deeper import is required. Did you fix this with `--fix`?

**Review by jkaster — APPROVED** [review]
LGTM. Other than the deeper import link (which I don't really care about), I like these prettier reformatting choices better.

File: packages/api-explorer/src/components/DocMethodSummary/utils.tsx (hunk 1 above)
Comment (bryans99) [reply packages/api-explorer/src/components/DocMethodSummary/utils.tsx:30]: Unfortunately not :( . That one was a bit painful

Comment (jkaster) [reply packages/api-explorer/src/components/DocMethodSummary/utils.tsx:30]: Ouch indeed

**Review by josephaxisa — APPROVED** [review]
lgtm

**Pull request merged**